src/filter.js

```diff
@@ -59,7 +67,16 @@ export class Filter extends CriteriaBuilder {
 
     //eslint-disable-next-line array-callback-return
     this.fieldElement.options.map(filter => {
-      this.fieldTypes[filter.name] = (filter.type === 'datetime') ? 'datetime-local' : filter.type;
+      this.fieldTypes[filter.value] = (filter.type === 'datetime') ? 'datetime-local' : filter.type;
+
+      if (filter.type === 'select') {
+        this.fieldEnumerations[filter.value] = (filter.options || []).map(option => {
+          return {
+            name : this.i18n.tr(option),
```

Comment:
@jeremyvergnas, aurelia form's select type uses the t= custom attribute on it's options. You could just pass option.name.